graph/resolver.go

```diff
@@ -17,25 +20,60 @@ import (
 //
 // It serves as dependency injection for your app, add any dependencies you require here.
 
+// Resolver holds the repositories for the graph resolvers.
```

Comment:
I can also make interfaces for those. I was not sure if I should only create interfaces for the ones that we currently need to mock.

As for where the definitions live, it is the convention to have interfaces defined by the consumer. Since they are the ones using the code. If you are curious a more in-depth explanation can be found here https://go.dev/wiki/CodeReviewComments#interfaces

Reply:
@Allyson-English 's suggestion is good, however i think we should stick to interfaces for what we are currently working on. We can either make a ticket for the other stuff or handle it as we work on it.